Exercise 2/Exercise2.rst

```diff
@@ -1,5 +1,9 @@
-Look at the file titled titled "Match This Output.png" in this directory.
+Basics of Cooking
+#################
+Boiling Water
+*************
```

Comment:
Stylistically, you'll need a blank line after header markings (such as `********`)